interface/forms/clinical_instructions/new.php

```diff
@@ -23,7 +23,7 @@
 // Author:   Jacob Paul <jacob@zhservices.com>
 //
 // +------------------------------------------------------------------------------+
-
+use OpenEMR\Core\Header;
```

Comment:
@robertdown ,
Is it better to place this use call above or below the include_once lines (or does it matter)?